hyperspy/tests/signal/test_remove_background.py

```diff
@@ -266,8 +266,10 @@ def compare_axes_manager_metadata(s0, s1):
 @pytest.mark.parametrize('show_progressbar', [True, False])
 @pytest.mark.parametrize('plot_remainder', [True, False])
 @pytest.mark.parametrize('background_type',
-                         ['Doniach', 'Gaussian', 'Lorentzian', 'Polynomial',
-                          'Power Law', 'Offset', 'SkewNormal', 'SplitVoigt',
+                         ['Doniach', 'Exponential', 'Gaussian', 'Lorentzian', 
+                          'Polynomial', 'Power law', 'Power Law', 'PowerLaw', 
+                          'Offset', 'Skew normal', 'Skew Normal', 'SkewNormal', 
+                          'Split Voigt', 'Split voigt', 'SplitVoigt',
```

Comment:
The number of combination for this test is getting unnecessarily large. Maybe the `background_type` parameter should be move to another test? The test suite is getting fairly slow, it is worth making sure that we don't add more test which are inefficient.

Reply:
I can move these variations to the specific tests for Power law, Skew normal and Split Voigt or create a dedicated tests only that these variations are accepted.